js/misc/parentalControlsManager.js

```diff
@@ -103,5 +103,19 @@ var ParentalControlsManager = class {
 
         return this._appFilter.is_appinfo_allowed(appInfo);
     }
+
+    isContentTypeAllowed (contentType) {
+        // Are parental controls disabled at runtime?
+        if (this._disabled)
+            return true;
+
+        // Have we finished initialising yet?
+        if (!this._appFilter) {
+            log('Warning: Hiding app because parental controls not yet initialised: ' + appInfo.get_id());
```

Comment:
`appInfo` is not defined. This is a copy-paste error from the block above.

Reply:
Addressed! Thanks